src/main/resources/site/parts/subjectArticleList/subjectArticleList.jsx

```diff
@@ -71,21 +71,25 @@ function SubjectArticleList(props) {
   }
 
   function renderArticles() {
-    return articles.map((article, i) => {
-      return (
-        <div key={i} className='mt-5'>
-          <Link href={article.url} className='ssb-link header'>
-            {article.title}
-          </Link>
-          <p>
-            <Truncate lines={2} className='article-list-ingress'>
-              {article.preface}
-            </Truncate>
-          </p>
-          <time dateTime={article.publishDate}>{article.publishDateHuman}</time>
-        </div>
-      )
-    })
+    return (
+      <ol className='list-unstyled'>
```

Comment:
This should be ul, this list is not numbered. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol